src/Resources/assets/controllers/datetime_controller.js

```diff
@@ -14,15 +16,15 @@ export default class extends Controller {
         const enableTime = type === 'time' || type === 'datetime-local';
         let plugins = [KbdPlugin];
 
-        if(enableTime) {
+        if (enableTime) {
             plugins.push(TimePlugin);
         }
 
         const picker = new easepick.create({
             element: this.element,
             css: easepickStyle,
             lang: this.langValue || 'de-DE',
-            format: enableTime ? "YYYY-MM-DDTHH:mm" : "YYYY-MM-DD",
+            format: this.formatValue || (enableTime ? "YYYY-MM-DDTHH:mm" : "YYYY-MM-DD"),
```

Comment:
Are we sure this doesn't «throw» without having either a `default` defined under `values` for the `format` value, or a `this.hasFormatValue` check? In my experience, Stimulus isn't very forgiving with that. Or wasn't, at least.

Reply:
No, all good. It returns an empty string in this case.

It's not set in the controller.
I want it to be optional and use the empty string to set the correct format according to a certain logic.